packages/perseus-core/src/analytics.ts

```diff
@@ -0,0 +1,26 @@
+import type {AnalyticsEvent} from "@khanacademy/event-schemas";
+
+// Magic based on:
+//   * https://echobind.com/post/slicing-typescript-literal-strings
+//   * https://www.typescriptlang.org/docs/handbook/utility-types.html
+//
+// This type removes keys that Perseus cannot fill in. It is expected that the
+// hosting application will populate these missing values. We represent this as
+// a type so taht the host can easily detect which fields in the type are
```

Comment:
`...so that the...`

Reply:
Removed this as I've moved away from depending on the CEDAR event schema because it's private (Matthew pointed this out).